en/pe-docs/docs/devops-and-ci-cd/manage-environments.md

```diff
@@ -0,0 +1,44 @@
+# Manage Environments
+
+By default, all projects created in the cloud data planes (irrespective of the data plane region) are provisioned with two environments (i.e., development and production).
+
+The environments are listed in the order of deployment and promotion. The initial deployment takes place in the first environment and you can proceed to promote a component to subsequent environments.
+
+## Create a new environment
+
+### Prerequisites
+
+- To create additional environments, you must have a Choreo subscription or a private data plane.
+- To create a new environment in a private data plane organization, you must have the `ENVIRONMENT-MANAGEMENT` permission. By default, `ENVIRONMENT-MANAGEMENT` permission is granted to Admin and Choreo DevOps roles.
+
+To create a new environment, follow the steps given below:
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/) and select the **Organization**.
+2. In the left navigation menu, click **Infrastructure** and then click **Environments** (note that this is the **Environments** page under your organization, not your projects).
```

Comment:
Shall we move this note to a tip? 
Existing place: https://wso2.com/choreo/docs/monitoring-and-insights/view-logs/#audit-logs

Reply:
we can remove the entire thing neh. This is not worth mentioning as a Note IMO. It says about simply the place you should look for.

Reply:
But you can view environments both in project and org level. So isn't it better to let them know that you can only create envs in org level?